client/src/app/site/pages/meetings/modules/poll/components/base-poll-vote/base-poll-vote.component.html

```diff
@@ -73,6 +73,11 @@ <h4 class="poll-delegation-title">
                                     'yna-grid':
                                         poll.isMethodYNA ||
                                         (settings.isSplitSingleOption && displayed_in_autopilot && poll.isMethodYNA),
+                                    'yna-grid-extended':
+                                        poll.isMethodYNA &&
+                                        isMobile &&
+                                        !settings.isSplitSingleOption &&
+                                        poll.options[0].isListOption,
```

Comment:
- please remove isSplitSingleOption 
- change poll.options... to poll.isListPoll
- Add !isMobile to yna-grid